client/analytics/components/report-filters/index.js

```diff
@@ -31,30 +30,19 @@ class ReportFilters extends Component {
 		this.onAdvancedFilterAction = this.onAdvancedFilterAction.bind( this );
 	}
 
-	showCesSurveyTrack() {
-		const { addCesSurveyTrack } = this.props;
-		addCesSurveyTrack(
-			'woocommerce_admin_analytics_filtered',
-			__(
-				'How easy was it to filter your store analytics?',
-				'woocommerce-admin'
-			)
-		);
-	}
-
 	onDateSelect( data ) {
-		const { report } = this.props;
-		this.showCesSurveyTrack();
+		const { report, addCesSurveyTrackForAnalytics } = this.props;
+		addCesSurveyTrackForAnalytics();
```

Comment:
I think you're going off track a bit here, the data store is meant to be reusable across many different consumers, not just for analytics. Can you make it generic?

Reply:
@becdetat That's true. For the consumers, there is a generic action [addCesSurveyTrack](https://github.com/woocommerce/woocommerce-admin/blob/8204aa31cf41f819b56943977a03b9519a345d29/client/customer-effort-score-tracks/data/actions.js#L30).

Reply:
Ok cool